src/index.html

```diff
@@ -486,7 +486,7 @@ <h3>Keep Talking and Nobody Explodes</h3>
 				switch (count.total) {
 					case 3: {
 						if (count.red == 0) {
-							result.join(cut.second);
+							result.push(cut.second);
```

Comment:
Could this be what's causing main to fail on `blue, yellow, yellow` but anibor to work?

Reply:
Yes. Fixed in main.